### PR TITLE
Lra 413 art survey design and add a database

### PR DESCRIPTION
**Pull request by britaumich**
Here is the art_survey database.
Some questions about the art_items table:
description - text (rich text) or string?
value_cost - integer? or float?

### CODE REVIEW

**Review by rsmoke — COMMENTED** [review]
I insist that updated_by be foreign key associations
The spacing is wonky should be 2 spaces not 4 spaces
the associations need to be expressed in the factories
reduce the seed file
...

File: app/models/annotation.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Table name: annotations
+#
+#  id         :bigint           not null, primary key
+#  uri        :string
+#  updated_by :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Annotation < ApplicationRecord
+	has_rich_text :note
```
[the remark this answers is not in the document]
Comment (britaumich) [reply app/models/annotation.rb]: spacing has been fixed

**Comment by britaumich** [comment]
I removed updated_by (string) with [reference :user] for all models;
I left some commands in the seed file but commented them. We can't run the seed file automatically now because a user should be created before these commands are run.

**Comment by britaumich** [comment]
I agree with closing this pull request and creating the database again without extra migrations.
But I don't agree with the permissions table design. A table with four fields doesn't need normalization. 
It will cause join multiple tables in queries that should be run at every click in the application.

I'll put a quote from [Normalization Vs Denormalization in Database Modeling](https://medium.com/analytics-vidhya/database-normalization-vs-denormalization-a42d211dd891) article here
“Normalization and denormalization are useful according to the situation. Normalization is used when the faster insertion, deletion and update anomalies, and data consistency are necessarily required. On the other hand, Denormalization is used when the faster search is more important and to optimize the read performance. It also lessens the overheads created by over-normalized data or complicated table joins.”

You can close this pul request.

**Pull request closed without merging**